### PR TITLE
VACMS-16909: Add setup for v2 of DUW

### PR DESCRIPTION
**Pull request by chriskim2311**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

Add setup for v2 of DUW, dynamic routes for prod to not show v2 pages. v2 will be built using the same root URL as the current Discharge Upgrade Wizard. I will separate out the code of v2 in folders but will share a few files. I will not be touching any code related to the existing wizard. All changes will be localized to dev and staging. 

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16909

## Testing done
Tested route logic locally.

## What areas of the site does it impact?
Discharge Upgrade Wizard

## Acceptance criteria
- [ ] A11y review
- [ ] Design review
- [ ] PM review

### CODE REVIEW

**Review by randimays — APPROVED** [review]

File: src/applications/discharge-wizard/components/v2/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+import { Link } from 'react-router';
+
+const HomePage = () => {
+  return (
+    <section
+      className="dw-instructions"
+      itemScope
+      itemType="http://schema.org/FAQPage"
+    >
+      <h1 itemProp="name">How to Apply for a Discharge Upgrade @@@@</h1>
```
Comment (chriskim2311) [inline src/applications/discharge-wizard/components/v2/Homepage.jsx:11]: To distinguish second homepage. Will remove.

**Comment by chriskim2311** [comment]
@laflannery @thejordanwood @FranECross   This PR is ready for your review. Just the setup of the new DUW introduction page and new routing logic.  It has the same content as the original page just a new route to build off of.

`/discharge-upgrade-instructions/introduction`

**Comment by laflannery** [comment]
@chriskim2311 I have a question - Is this wizard going to change to the subtask pattern (one-thing-per-page) - like we are using on the PACT Act?

**Comment by FranECross** [comment]
@laflannery Popping in to answer your question. It is going to change to the subtask pattern. :)

**Comment by chriskim2311** [comment]
@laflannery Yup! Next PR will have more functionality and will be the start of the sub-task pattern. The button goes to the original flow currently but will change

**Pull request merged**